### PR TITLE
FIXIT summit - Fix lint warnings

### PR DESCRIPTION
**Pull request by dparikh**
Issue #373

### CODE REVIEW

File: src/api/user-data.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ export class UserData {
     /** @const {string} */
     this.idToken = idToken;
     /** @private @const {!Object} */
-    this.data = data;
+    this.data_ = data;
```
Comment (dvoytenko) [inline src/api/user-data.js]: Please keep the name and make it public instead.

File: src/utils/jwt.js [hunk 2]
```diff
@@ -33,7 +33,9 @@ let JwtTokenInternalDef;
  * Provides helper methods to decode and verify JWT tokens.
  */
 export class JwtHelper {
-
+  /**
```
Comment (dvoytenko) [inline src/utils/jwt.js]: Remove this one. Really not a big fun of generic comments.

File: src/api/user-data.js (hunk 1 above)
Comment (dparikh) [reply src/api/user-data.js]: Done

File: src/utils/jwt.js (hunk 2 above)
Comment (dparikh) [reply src/utils/jwt.js]: Done

**Pull request merged**